client-js/app/elements/labrad-registry.ts

```diff
@@ -42,15 +43,76 @@ export class LabradRegistry extends polymer.Base {
                                               this.$.editValueInput);
   }
 
+
+  getDefaultSelectedItem(): number {
+    return this.path.length > 0 ? 1 : 0;
+  }
+
+
+  cursorMove(e) {
+    const offset = this.path.length > 0 ? 1 : 0;
```

Comment:
This isn't repetition, they are two bits of code that do the same thing for different reasons.

The offset is always. It just so happens that we've currently decided that the default selected item is also the first item after the offset. However, `getDefaultSelectedItem` could be changed to `return 0;` which would be an appropriate default, but would break the offset.

I can fork them both from a "hasPath" function, but that seems a little silly.